scrapi/base/__init__.py

```diff
@@ -66,13 +75,45 @@ class OAIHarvester(BaseHarvester):
     record_encoding = None
 
     def __init__(self, name, base_url, timezone_granularity=False, timeout=0.5, property_list=None, approved_sets=None):
-        self.name = name
+        self.NAME = name
         self.base_url = base_url
         self.property_list = property_list or ['date', 'language', 'type']
         self.approved_sets = approved_sets
         self.timeout = timeout
         self.timezone_granularity = timezone_granularity
 
+    @property
+    def name(self):
+        return self.NAME
+
+    @property
+    def namespaces(self):
+        return self.NAMESPACES
+
+    @property
+    def schema(self):
+        properties = {
+            'properties': {
+                item: (
+                    '//dc:{}/node()'.format(item),
+                    '//ns0:{}/node()'.format(item),
+                    self.resolve_property
+                ) for item in self.property_list
+            }
+        }
+        return update_schema(OAISCHEMA, properties)
+
+    def resolve_property(self, dc, ns0):
```

Comment:
dc corresponds to the dc namespace, ns0 to the ns0 namespace. What would you prefer they be named?